notebooks/204-vision-facies-segmentation/204-vision-facies-segmentation.ipynb

```diff
@@ -0,0 +1,775 @@
+    "            out = np.argmax(out, axis=1).squeeze()\n",
+    "            predicted_cube[slice_index, :, :] = out\n",
+    "        if index_of_dim == 1:\n",
+    "            inp = data[:, slice_index, :]\n",
+    "            out = exec_net.infer(inputs={\"input\": inp})[\"output\"]\n",
+    "            out = np.argmax(out, axis=1).squeeze()\n",
+    "            predicted_cube[:, slice_index, :] = out\n",
+    "        if index_of_dim == 2:\n",
+    "            inp = data[:, :, slice_index]\n",
+    "            out = exec_net.infer(inputs={\"input\": inp})[\"output\"]\n",
+    "            out = np.argmax(out, axis=1).squeeze()\n",
+    "            predicted_cube[:, :, slice_index] = out\n",
+    "    return predicted_cube"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def discrete_cmap(N, base_cmap=None):\n",
+    "    \"\"\"Create an N-bin discrete colormap from the specified input map\"\"\"\n",
+    "\n",
+    "    # Note that if base_cmap is a string or None, you can simply do\n",
+    "    #    return plt.cm.get_cmap(base_cmap, N)\n",
+    "    # The following works for string, None, or a colormap instance:\n",
+    "\n",
+    "    base = plt.cm.get_cmap(base_cmap)\n",
+    "    color_list = base(np.linspace(0, 1, N))\n",
+    "    cmap_name = base.name + str(N)\n",
+    "    return base.from_list(cmap_name, color_list, N)\n",
+    "\n",
+    "\n",
+    "def show_legend(N, cmap_name):\n",
+    "    base = plt.cm.get_cmap(cmap_name)\n",
+    "    color_list = base(np.linspace(0, 1, N))\n",
+    "    print(color_list)\n",
+    "\n",
+    "\n",
+    "def show_legend(labels, cmap):\n",
+    "    N = len(labels)\n",
+    "    fig = plt.figure(figsize=(12, 6))\n",
+    "    ax1 = fig.add_axes([0.05, 0.80, 0.9, 0.15])\n",
+    "    cb1 = mpl.colorbar.ColorbarBase(\n",
+    "        ax1,\n",
+    "        cmap=cmap,\n",
+    "        ticks=np.arange(0, N, 1) / N + 1 / (2 * N),\n",
+    "        orientation=\"horizontal\",\n",
+    "    )\n",
+    "    cb1.ax.set_xticklabels(labels, fontsize=20)\n",
+    "    cb1.set_label(\"Legend\", fontsize=24)\n",
+    "    plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Get config and load dataset"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "config = get_config()\n",
+    "data = load_data(config)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Running"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Load model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ie = IECore()\n",
+    "ie.add_extension(get_extensions_path(), \"CPU\")\n",
+    "net = ie.read_network(config[\"model\"])"
+   ]
+  },
```

Comment:
@alexeyhorkin Thanks for your contribution to the notebooks. I have tried the notebooks and I have an issue with openvino-extensions. 
![image](https://user-images.githubusercontent.com/10940214/159775747-2963992a-07ae-4cbf-abe5-fe55323e466c.png)

Reply:
@paularamo thanks for testing. Actually I know about it, so it's still WIP